### ISSUE TITLE
Add example with GPU and mini-batch gradient descent

### ISSUE THREAD

**Issue by AlexGose** [AlexGose/learning-curve#9]
This feature will show how to plot a learning curve for a Pytorch model trained with the MNIST data set.  Training will be done with GPU processing and mini-batch gradient descent.
